include/Util/SVFUtil.h

```diff
@@ -114,17 +114,10 @@ inline bool cmpPts (const PointsTo& lpts,const PointsTo& rpts)
 }
 
 
-/// Return true if this function is llvm dbg intrinsic function/instruction
-//@{
-inline bool isIntrinsicDbgFun(const Function* fun)
-{
-    return fun->getName().startswith("llvm.dbg.declare") ||
-           fun->getName().startswith("llvm.dbg.value");
-}
-/// Return true if it is an intric debug instruction
-inline bool isInstrinsicDbgInst(const Instruction* inst)
+/// Return true if it is an intrinsic instruction
+inline bool isIntrinsicInst(const Instruction* inst)
 {
-    return SVFUtil::isa<llvm::DbgInfoIntrinsic>(inst);
```

Comment:
What is the reason for changing to IntrinsicInst? It seems to me that we can have two functions here?